nautobot/docs/development/core/getting-started.md

```diff
@@ -714,8 +712,6 @@ Integration tests are run using the `invoke integration-test` command. All integ
 !!! tip
     You may also use `invoke integration-test` in the Virtual Environment workflow given that the `selenium` container is running, and that the `INVOKE_NAUTOBOT_LOCAL=True` environment variable has been set.
 
-+/- 1.5.11
-    The `--cache-test-fixtures` argument was added to the `invoke integration-test` and `nautobot-server test` commands to allow for caching of test factory data between test runs. See the [factories documentation](./testing.md#factory-caching) for more information.
```

Comment:
Maybe change this to another `!!! tip` annotation instead of removing it?

Reply:
Sounds good, I made a tip for each of those two statements.